source/src/commands/onSelectMenu.ts

```diff
@@ -1,9 +1,41 @@
 // The module 'vscode' contains the VS Code extensibility API
 // Import the module and reference it with the alias vscode in your code below
 import * as vscode from 'vscode';
-
 export default vscode.commands.registerCommand('celerik-scaffolder.onOpenMenu', () => {
-    // The code you place here will be executed every time your command is executed
-    // Display a message box to the user
-    vscode.window.showInformationMessage('Hello VS Code!');
+    const panel = vscode.window.createWebviewPanel(
+        'celerikScaffolder', // Identifies the type of the webview. Used internally
+        'Celerik Scaffolder', // Title of the panel displayed to the user
+        vscode.ViewColumn.One, // Editor column to show the new webview panel in.
+        {} // Webview options. More on these later.
+    );
+
+    panel.webview.html = getWebviewContent(); // set HTML content
 });
+
+function getWebviewContent() {

```

Comment:
shouldn't the getWebviewContent function be the result of the build

Reply:
No, in this case getWebviewContent only contains the html that will be mapped when using vsCode's createWebviewPanel function. We are just separating this from the rest of the function to keep the code a bit more readable. The mapping is done on line 12